### ISSUE TITLE
Главная. Форма обратной связи. Доступен для ввода некорректный email

### ISSUE THREAD

**Issue by orlovrs** [MailTestingProject/CarsMailTest#8]
Описание проблемы
Форма обратной связи пропускает адреса, которые не могут существовать

Шаги для повторения
1. Перейти на главную страницу проекта https://cars.mail.ru/
2. Нажать на кнопку "Напишите нам!"
3. Ввести в поле адреса "234234@345345.rs"
4. Нажать на кнопку "Отправить"

Ожидали
Сообщение об ошибке, что адрес некорректен, поле подсвечено красным

Получили
Адрес успешно обрабатывается

Слишком длинные email адреса так же проходят

**Comment by emukhin** [comment]
могу ошибаться, но мне видится адрес 234234@345345.rs вполне валидным. что с ним не так?

**Comment by orlovrs** [comment]
Нет, не ошибаешься... мой косяк, что-то в голове в тот момент перекрутило, видимо
